ansys/mapdl/core/inline_functions.py

```diff
@@ -0,0 +1,367 @@
+
+
+class _ParameterParsing:
+    def _parse_parameter_integer_response(self, response) -> int:
+        return int(self._parse_parameter_float_response(response))
+
+    @staticmethod
+    def _parse_parameter_float_response(response) -> float:
+        parts = response.split('=')
+        if len(parts) != 2:
+            raise TypeError('Expression not provided; parameter response not recognised.')
+        number = parts[1].strip()
+        return float(number)
+
+
+class _ComponentQueries(_ParameterParsing):
+    _mapdl = None
+
+    def centrx(self, e: int) -> float:
+        """
+        Fetches centroid X-coordinate of element ``e`` in global
+        Cartesian coordinate system. Centroid is determined from the
+        selected nodes on the element.
+
+        Parameters
+        ----------
+        e : int
+            The element number of the element to be considered.
+
+        Returns
+        -------
+        number : float
+            The centroid coordinate.
+
+        Examples
+        --------
+        Here we construct a line between the coordinates (0, 0, 0) and
+        (1, 2, 3) then find the centroid x-coordinate of this element.
+
+        >>> from ansys.mapdl.core.inline_functions import Query
+        >>> from ansys.mapdl.core import launch_mapdl
+        >>> mapdl = launch_mapdl()
+        >>> mapdl.prep7()
+        >>> n0 = mapdl.n(1, 0, 0, 0)
+        >>> n1 = mapdl.n(2, 1, 2, 3)
+        >>> mapdl.et(1, 'LINK11')
+        >>> e0 = mapdl.e(n0, n1)
+        >>> q = Query(mapdl)
+        >>> q.centrx(e0)
+        0.5
+        """
+        response = self._mapdl.run(f'_=CENTRX({e})')
+        return self._parse_parameter_float_response(response)
+
+    def centry(self, e: int) -> float:
+        """
+        Fetches centroid Y-coordinate of element ``e`` in global
+        Cartesian coordinate system. Centroid is determined from the
+        selected nodes on the element.
+
+        Parameters
+        ----------
+        e : int
+            The element number of the element to be considered.
+
+        Returns
+        -------
+        number : float
```

Comment:
I don't do this yet, but let's start writing our returns as:
```
Returns
--------
float
    The y centroid coordinate.
```

Reply:
will this work with sphinx?

Reply:
Done!